specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/cloudService.json

```diff
@@ -106,7 +106,8 @@
             "description": "The expand expression to apply to the operation.",
             "type": "string",
             "enum": [
-              "instanceView"
+              "instanceView",
+              "userData"
```

Comment:
Cloudservice doesn't support UserData

Reply:
Yes, thats true. (I need to document that somewhere, but waiting for Sameer to confirm where that documentation should go)
However, in order to add a new enum value to InstanceViewTypes (or any other existing enum), the enum value has to be added to all instances of InstanceViewTypes.
Otherwise, the PR check and AutoRestGeneration will fail with this error

“Swagger document contains two or more x-ms-enum extensions with the same name 'InstanceViewTypes' and different values: instanceView,userData vs. instanceView”

Reply:
The name _InstanceViewTypes_ doesn't accurately describe the accepted values anymore though.

Reply:
Yes I agree, and that is very unfortunate.
The dev who created 'InstanceViewTypes' should instead have used a naming pattern like  'ExpandTypesFor{resourceName}' (e.g. 'ExpandTypesForVirtualMachine' or 'ExpandTypesForVirtualMachineScaleSet'), which would have allowed future features to extend the $expand for different scenarios.

Unfortunately, we are now stuck with this terrible name 'InstanceViewTypes'.
Using a new enum would be a breaking change.
Since we have a work-around (adding new enum value to existing enum), BreakingChange review board says that we should use the workaround instead of breaking our customers.